src/Messenger.Client/Services/IMessengerMessageSender.cs

```diff
@@ -9,5 +9,9 @@ public interface IMessengerMessageSender
         Task<MessengerResponse> SendAsync(MessengerMessage message, MessengerUser recipient);
 
         Task<MessengerResponse> SendAsync(MessengerMessage message, MessengerUser recipient, String accessToken);
+
```

Comment:
I think it makes sense to move them to a separate interface to not to mix with messages.

Reply:
I modelled this after the messenger API where sender actions are actually part of the message API (https://developers.facebook.com/docs/messenger-platform/reference/send-api). As you can see 'message' and 'sender_action' are simply different attributes of the same end-point – even though they are indeed mutually exclusive. Perhaps this is an arch smell that they should be separate end-points on their side, but we should probably stick to their abstraction since the client library should be a close reflection of their API. We could perhaps rename 'IMessengerMessageSender' to something more general to reflect the broader 'Send API' scope ('IMessengerSendService'?)